data_processing/job_event.py

```diff
@@ -0,0 +1,149 @@
+"""Referenced from bot.py and filters and returns
+jobs that match the inputted criteria.
+"""
+
+from typing import Any
+from data_collections.csv_updater import (
+    extract_entries_from_csv,
+)
+
+
+def paste_jobs_command(command_args: str) -> str:
+    """
+    Parses the jobs command arguments to extract search filters
+        from flag notation
+
+    Args:
+        command_args (str): The arguments passed to the command,
+                                as a string.
+
+    Returns:
+        str: string containing parsed parameters
+    """
+    params = ""
+    if not command_args:
+        return params
+    args = command_args.split()
+    for term in args:
+        if term.startswith("-"):
+            pass
+        else:
+            params += term + " "
+    return params.strip()
```

Comment:
@dalawton Gotcha. 

Your jobs works as intended, but now this method starts to seem arbitrary. It's looking for a flag indicator, but we never give any instruction to the user to input a flag.
```suggestion
def paste_jobs_command(command_args: str) -> str:
    """
    Returns the command line arguments exactly as passed in,

    Args:
        command_args (str): The arguments passed to the command.

    Returns:
        str: The arguments string as-is, stripped of leading/trailing whitespace.
    """
    if not command_args:
        return ""
    return command_args.strip()
```

You can also remove this method and move the logic into `bot.py`